horovod/spark/torch/remote.py

```diff
@@ -194,6 +195,7 @@ def train(serialized_model, optimizer_cls, model_opt_state_serialized,
             # Pass the compression arg only if it is specified by the user.
             dist_optimizer_args['compression'] = gradient_compression
         # Horovod: wrap optimizer with DistributedOptimizer.
+        dist_optimizer_args['backward_passes_per_step'] = backward_passes_per_step
```

Comment:
I just realized both torch distributed optimizer and this estimator don't check the value for this. Could you add checks in those places to make sure they are always > 0? Since you are doing division with this number, we need to careful.